frontend/src/metabase/dashboard/components/DashboardTabs/DashboardTabs.tsx

```diff
@@ -75,7 +78,12 @@ export function DashboardTabs({
   }
 
   return (
-    <Flex align="start" gap="lg" w="100%" className={className}>
+    <Flex
+      align="start"
+      gap="lg"
+      w="100%"
+      className={cx(S.dashboardTabs, isNightMode && S.isNightMode, className)}
```

Comment:
I don't see much usage of manual class condition with `cx`. searching regex `cx\(.*&&` returns 4 (5 including this place) results.

Can you use the object syntax that has been more established?
e.g.
https://github.com/metabase/metabase/blob/0831ff0d708a56a20f5c7e0e28f798496c65ab8f/frontend/src/metabase/admin/settings/components/widgets/SettingInput/SettingInput.tsx#L56-L59